src/javascript/utils/compose.js

```diff
@@ -0,0 +1 @@
+export const compose = (...funcs) => funcs.reduce((a, b) => (...args) => a(b(...args)), arg => arg);
```

Comment:
are we not using this in other projects too? so should we not put this code somewhere we can share it?

Reply:
Were you want to put it ? data-helper is not an option, as we already discuss.

I think we can keep this here, as we will progressively delete HOC pattern for use hooks.